src/patternfly/components/HelperText/helper-text.scss

```diff
@@ -45,29 +50,35 @@
 
 .#{$helper-text}__item {
   display: flex;
+  font-weight: var(--#{$helper-text}__item--FontWeight);
```

Comment:
While I doubt this would be an issue... I think this belongs on `__item-text` since the text is what we want bolded. Say we added something like sub-text later down the road, or a plain action menu, those and any of their children would be bold from this declaration when we probably don't want that as the default.

Reply:
Updated these lines to add the __item-text and removed the unused text-color stuff